src/ray/util/process.cc

```diff
@@ -629,6 +629,8 @@ bool IsProcessAlive(pid_t pid) {
   }
   return false;
 #else
+  // Note if the process is a zombie (dead but not yet reaped), it will
+  // still be alive by this check.
```

Comment:
We still kill them inside KillChildProcs right?

Reply:
Yep, all child procs are sent kill signal regardless of process state